src/routes/rules/RulesView.vue

```diff
@@ -161,129 +154,99 @@
         <p class="mb-4">
-          Yes! Queens protect against targeting, when a card applies a unique effect to one specific card.
-          This means your other cards (not the Queen, herself) are protected against Twos (either effect),
-          Nines, and Jacks.
+          {{ `${t('rules.queenProtectTwoAnswer')}` }}
         </p>
-        <h3>Do Queens protect against scuttling? What about Aces & Sixes?</h3>
+        <h3>{{ `${t('rules.queenProtectScuttle')}` }}</h3>
         <p class="mb-4">
-          No! Scuttling doesn't count as targeting, and neither do the board-wiping effects of Aces or Sixes.
-          Queens only protect against the effects of Twos, Nines, and Jacks.
+          {{ `${t('rules.queenProtectScuttleAnswer')}` }}
         </p>
-        <h3>Can I counter a Two with a Two?</h3>
+        <h3>{{ `${t('rules.twoOnTwo')}` }}</h3>
         <p class="mb-4">
-          Heck yeah! Playing a Two to counter is a One-Off, which can be countered with a two. Stacked
-          counters make for exciting plays and pivotal strategic moments.
+          {{ `${t('rules.twoOnTwoAnswer')}` }}
         </p>
-        <h3>Can I win by playing a King?</h3>
+        <h3>{{ `${t('rules.kingWin')}` }}</h3>
         <p class="mb-4">
-          Yes! If you meet the required number of points, you win immediately. This means if you play a King
-          when you already have enough points to meet the new limit, you win on that turn.
+          {{ `${t('rules.kingWinAnswer')}` }}
         </p>
-        <h3>If I play an Ace or a Six, are my cards destroyed as well?</h3>
+        <h3>{{ `${t('rules.aceDestruction')}` }}</h3>
         <p class="mb-4">
-          Yes. Aces and Sixes destroy all Point Cards and all Royals plus Glasses Eights, respectively. That
-          includes any that you have out. Try to avoid destroying too many of your own cards!
+          {{ `${t('rules.aceDestructionAnswer')}` }}
         </p>
-        <h3>My Seven revealed two Jacks and my opponent has no points. What happens?</h3>
+        <h3>{{ `${t('rules.revealNoPoints')}` }}</h3>
         <p class="mb-4">
-          If playing both cards revealed by a Seven is impossible (namely revealing two Jacks and your
-          opponent has no point cards in play), choose one card to discard and place the other back on top of
-          the deck.
+          {{ `${t('rules.revealNoPointsAnswer')}` }}
         </p>
-        <h3>What if I play a Seven and there is only one card left in the deck?</h3>
+        <h3>{{ `${t('rules.revealOneLeft')}` }}</h3>
         <p class="mb-4">
-          Reveal just the one card and continue play like normal. Similar to the above situation, if you
-          reveal a Jack and your opponent does not have any point cards in play, that Jack is scrapped.
+          {{ `${t('rules.revealOneLeftAnswer')}` }}
         </p>
-        <h3>What happens when the deck is exhausted?</h3>
+        <h3>{{ `${t('rules.deckExhaust')}` }}</h3>
```

Comment:
```suggestion
        <h3>{{  t('rules.deckExhaust')  }}</h3>
```

Reply:
You should be able to do it like this for all of them.

Reply:
Will do, thanks

Reply:
Resolved in commit 0584c1c